_episodes/41-code-review.md

```diff
@@ -373,17 +206,212 @@ you have to do the following:
+>> for some data where the variance and standard deviation are different.
+>> In more complex examples, it is often easier to spot code that looks like it could  be wrong
+>> and think of a test that will exercise it. This saves embarrassment if the code turns out
+>> to be right, means you have the missing test written if it is wrong, and is often quicker
+>> than trying to execute the code in your head to find out if it is correct.
+>>
+> {: .solution}
+{: .challenge}
+
+### What not to look for
+
+The overriding priority for reviewing code should be making sure progress is being made -
+don't let perfect be the enemy of good here. Further, research has shown that the first hour
+of reviewing code is the most effective, with diminishing returns after that.
+
+To that end, here are a few things you shouldn't be trying to spot when reviewing:
+
+#### Linting issues, or anything else that an automated tool can spot
+
+Get the CI to do this - this will save the reviewer time, be more accurate
+and avoid needless conflict.
+
+#### Bugs
+
+It is easier to make sure there are sufficient tests - you are not an accurate
+computer simulator anyway.
+
+#### Issues that pre-date the change
+
+You may spot something that the reviewer didn't introduce and think they could
+fix it while in the area, but this can be a rabbit hole. A better approach would be to
+raise a PR after this one has been merged fixing the thing you spotted.
+
+### Responding to review comments
+
+When you receive comments on your review, there are a few different things that you will want
+to do.
+
+With some, you will understand and agree with what the reviewer is saying. With these comments,
+you should make the change to your code on your branch. Once you've made the change you can
+commit it. It might be helpful to add a thumbs up reaction to the comment, so the reviewer knows
+you have addressed it.
+
+TODO: screenshot of adding a emoji reaction.
+
+With some, the comment might not make total sense. You can reply to comments for clarification.
+
+TODO: screenshot of adding a comment.
+
+However, if you disagree, or are really lost on what they are driving it, it will be best to
+talk to them in person. Discussions done on code reviews can often feel quite adversarial -
+discussing what the best solution is in person can often defuse this.
+
+> ## Exercise: responding and addressing comments
+>
+> Look at the PR that you created on your repo, that should now have someone elses comments
+> on it.
+> For each comment, either reply explaining why you don't think the change is necessary
+> or make the change and push a commit fixing it. You can reply to the comment indicating you
+> have done it.
+>
+> At the same time, people will be addressing your comments. If you're happy that your
+> comment has been suitably addressed, you can mark it as resolved. Once you're happy they
+> have all been addressed, you can approve the PR.
+{: .challenge}
 
 #### Step 6: Closing a Pull Request
```

Comment:
Will need to get rid of "Step 6" and make sure the heading level is in line with the above.